### PR TITLE
Fix multiline haddocks in indented contexts

### PR DESCRIPTION
**Pull request by brandon-leapyear**
Fixes https://github.com/parsonsmatt/fourmolu/issues/63 
Fixes https://github.com/parsonsmatt/fourmolu/issues/53

### CODE REVIEW

File: src/Ormolu/Printer/Meat/Common.hs [hunk 1]
```diff
@@ -171,11 +172,13 @@ p_hsDocString hstyle needsNewline (L l str) = do
           Asterisk n -> " " <> T.replicate n "*"
           Named name -> " $" <> T.pack name
         sequence_ $ intersperse (newline >> s) $ map txt' docLines
+
```
Comment (georgefst) [inline src/Ormolu/Printer/Meat/Common.hs:175]: I agree this is nicer with a few more line breaks. But I'm very weary of doing anything which unnecessarily increases the diff with upstream. Merging a new Ormolu version can be quite hairy already.

**Review by georgefst — CHANGES_REQUESTED** [review]
This looks great. Thanks!

Just the minor quibble on whitespace.

File: src/Ormolu/Printer/Meat/Common.hs (hunk 1 above)
Comment (brandon-leapyear) [reply src/Ormolu/Printer/Meat/Common.hs:175]: oof. completely understand. will revert

**Comment by brandon-leapyear** [comment]
@georgefst is this fix worth a patch release to hackage?

**Comment by georgefst** [comment]
We should hopefully be able to batch this up with a few other upcoming changes. Unless you could really do with it on Hackage for any reason?

**Comment by brandon-leapyear** [comment]
If there are upcoming changes, then it can wait. As long as there's some release within a few weeks?

**Comment by gustavoavena** [comment]
Is there an ETA for when this fix will be released?
Thanks!

**Comment by brandon-leapyear** [comment]
@georgefst any updates on a Hackage release?

**Pull request merged**